### ISSUE TITLE
OBS Repository does not contain any data

### ISSUE THREAD

**Issue by networkException** [ungoogled-software/ungoogled-chromium-archlinux#135]
See https://software.opensuse.org//download.html?project=home%3Aungoogled_chromium&package=ungoogled-chromium-arch

CC: @braewoods

**Comment by networkException** [comment]
aaaand it did not persist, sorry for the ping

**Comment by networkException** [comment]
This is occuring for longer now. @braewoods any clue?

**Comment by braewoods** [comment]
Yes, it's broken. It's why I had to manually write the instructions for that. AFAIK it doesn't impact the actual repository so I just ignored it.

**Comment by braewoods** [comment]
No. I'm referring to the repository instructions OBS would normally generate.
